elm/src/Faq/View.elm

```diff
@@ -0,0 +1,445 @@
+                    ]
+                ]
+            , row
+                [ width fill
+                , height fill
+                , paddingXY 10 10
+                , spacingXY 20 20
+                , Border.color colors.white2
+                , Border.rounded 3
+                , Border.shadow
+                    { offset = ( 0, 0 )
+                    , size = 2
+                    , blur = 8
+                    , color = colors.gray3
+                    }
+                , height <| minimum 500 fill
+                , width <| maximum 700 fill
+                , centerX
+                ]
+                [ column
+                    [ width fill
+                    , spacingXY 0 25
+                    , paddingXY 0 40
+                    ]
+                    [ row
+                        (heading
+                            ++ [ centerX
+                               , Font.size 30
+                               , height <| minimum 80 fill
+                               , Font.color colors.blue1
+                               ]
+                        )
+                        [ text copy.talentExpertsHead ]
+                    , row [ width <| maximum 500 fill, centerX ]
+                        [ paragraph
+                            (Styles.paragraph
+                                ++ [ Font.center
+                                   , paddingEach { left = 40, right = 40, top = 0, bottom = 30 }
+                                   , height <| minimum 120 fill
+                                   ]
+                            )
+                            [ text copy.talentExperts ]
+                        ]
+                    , link
+                        (Styles.paragraph
+                            ++ buttons.primary
+                            ++ [ width <| maximum 300 fill
+                               , Font.color colors.white3
+                               , centerX
+                               , Font.center
+                               , mouseOver [ Background.color colors.blue1 ]
+                               ]
+                        )
+                        { url = "mailto:talent@withflint.com", label = text "talent@withflint.com" }
+                    , link
+                        (Styles.paragraph
+                            ++ buttons.secondary
+                            ++ [ width <| maximum 300 fill
+                               , Font.color colors.white3
+                               , centerX
+                               , Font.center
+                               , mouseOver [ Background.color colors.gray1 ]
+                               ]
+                        )
+                        { url = "tel:+1 (844) 677-1180", label = text "+1 (844) 677-1180" }
+                    ]
+                ]
+            ]
+        ]
+    ]
+
+
+desktopLayout : Model -> List (Element Msg)
+desktopLayout model =
+    [ column
```

Comment:
I feel the code is too dense here, maybe abstract some rows and columns as functions?

Reply:
The style is not the same as the one posted on the [issue page](https://github.com/withflint/withflint.com/issues/1)